lib/vtls/vtls.c

```diff
@@ -609,6 +614,9 @@ CURLcode Curl_ssl_addsessionid(struct Curl_easy *data,
     return CURLE_OUT_OF_MEMORY;
   }
 
+  if(added)
+    *added = TRUE;
+
   DEBUGF(infof(data, "Added Session ID to cache for %s://%s:%d [%s]",
                store->scheme, store->name, store->remote_port,
                isProxy ? "PROXY" : "server"));
```

Comment:
This debug output should perhaps also only be output if indeed the session id was added?

Reply:
That is already implemented. If the session id was not added, "return" is called before this DEBUGF/infof statement.

Reply:
Aha, so just me not reading correctly. :+1: